mozip-server/mozip-api/src/main/java/org/nexters/mozipmozip/resume/controller/ResumeController.java

```diff
@@ -76,4 +77,9 @@ public ResponseEntity modifyResume(@RequestBody @Valid ResumeUpdateDto resumeUpd
         return ResponseEntity.ok().body(resumeService.save(resumeUpdateDto.of()));
     }
 
+    @PatchMapping("/state")
```

Comment:
요거 그냥 다시 보면서 생각한건데, 아래와 같은 건 어떨까?

```suggestion
    @PatchMapping("/{id}/state")
```

`/api/v1/resumes/{id}/state`가 좀더 자원에 대해 명확하게 표현하는 거 아닐까 싶어서